package.json

```diff
@@ -28,25 +28,25 @@
     "model"
   ],
   "dependencies": {
-    "@hapi/hoek": "^9.0.0",
+    "@hapi/hoek": "^11.0.2",
     "@hapipal/toys": "^3.2.0",
-    "joi": "^17.0.0"
+    "joi": "^17.8.3"
   },
   "peerDependencies": {
-    "@hapi/hapi": ">=19 <22",
+    "@hapi/hapi": ">=20 <22",
     "joi": ">=17 <18",
-    "knex": ">=0.16",
-    "objection": ">=1 <4"
+    "knex": ">=0.21.19",
+    "objection": ">=2 <4"
```

Comment:
Objection v3 is well over a year old, and it doesn't seem like v2 is going to receive updates anymore.  Similar goes for knex— v1 came out over a year ago and v2 is actually out!  If we're making a breaking change here, there would be something sort of nice/clean about supporting knex v0.95+ and objection v3+.  Any thoughts/feels?

Reply:
Happy to bump both of those up.  Also, since joi is at `>=17 <18` will just simplify that down to `^17.0.0`